### DISCUSSION TITLE
NOVA 4 (UPF) classification buckets too many products to quickly make a decision

### DISCUSSION THREAD

**Discussion by MarByteBeep** [openfoodfacts/openfoodfacts-server#12092] [Q&A]
### Problem
In a perfectly balanced world, supermarkets would carry just a few UPF products. The problem is that we are not living in a balanced world. A very high percentage of all foods in the supermarkets is categorized as NOVA 4: UPF. I believe over 80% in the UK stores is considered UPF(!) That means that 80% of all food (millions of products) is bucketed into a single category, which is very **not** ideal.

Obviously it would be best if supermarkets would change. They won't anytime soon. But the information an app presents can change!

For me as a customer this makes it difficult to make a decision on which UPF is _less worse_ than others, since all UPFs are bucketed in a single classification. Aside from the differences in amount of UPF ingredients, there is also no distinction between the types of UPF ingredient. There is no distinction at all between a veggie burger that has a single UPF ingredient added (e.g., vitamin B12) , and a product that has been glued together using 30 different UPF ingredients. I'm not a scientist, but I'm pretty sure there is a difference between the two.

As a customer the app isn't really helpful if most foods are labeled UPF. I find myself now manually looking into the different ingredients and make my decision based on that. This is cumbersome and I often don't know anything about those chemical names that are listed, making my decision even harder. So it would be very helpful if the app quickly provides me with that info. 

I afraid if people constantly see UPF for mosts foods, with no further distinction AND not a lot of alternatives, the app will lose its usefulness.

### Proposed solution
This might perhaps be more of an acadamic question, but it would be very helpful to subdivide category 4 from not good to very bad, or present it as some scale (see image below). At least until the producers drastically reduce the number of UPF ingredients (which will probably take a long time).

![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/58225562/63fa6116-04eb-4269-ad42-bb3ea1bbba35)

**Comment by monsieurtanuki** [comment]
Hi @MarByteBeep!

I'm not sure vitamin B12 is considered UPF.
For instance https://uk.openfoodfacts.org/product/5411188130031/plant-protein-alpro#panel_nova shows that the product is considered UPF but not because of the vitamin B12.

Fun fact: yesterday my sister and I were talking about food (we're French) and she was horrified by what she saw in the UK supermarkets, where she found predominantly what she described as "food for kids with agressive packaging" (implicitly UPF).
Therefore it may not make sense to extrapolate the whole world food is 80% UPF.

Your question is still relevant for UK, though.

Perhaps, instead of tap dancing around how horrible that food is compared to that other horrible food, we at openfoodfacts should be better at _suggesting alternate food_ that do match your criteria and with same taste/food category as an initial product.
Your "veggie burger" example may be really interesting, because a UPF "meat-like" soy burger could lead to a NOVA 3 spinach patty.

**Comment by MarByteBeep** [comment]
Hi there! 

Unfortunately it's not just the UK; over here in the Netherlands, supermarkets are loaded with UPF food as well. And yes, I'm with you on that suggestion alternative, but especially when being a vegetarian or vegan, it's very hard to find non UPF food. In that case the app will not help me since all veggy patties in my local supermarket are UPF. There is no alternative. Yet I still want to have a meat alternative, so in that case I still need to know the differences between these foods.

E.g., compare this one

https://nl.openfoodfacts.org/product/8720182202062/kipstuckjes-de-vegetarische-slager#panel_nova

(with only 1 ingredient causing it to end up with a UPF label: the vague 'aroma', why is that considered UPF might be an interesting conversation on its own)

with this one:

https://nl.openfoodfacts.org/product/8718300883040/plantaardige-balletjes-vivera#panel_nova

That also has 'aroma' but in addition 5 more additives. But unless clicking multiple layers deeper in the app, as a customer don't know that, I only see "ultra processed foods" for both.

Adding some sort indicator as proposed above, would at least solve this until we see way more healthy alternatives in the supermarkets.

**Comment by monsieurtanuki** [comment]
@MarByteBeep What if you could have a quicker access to Nova data, as expanded, instead of having to scroll down to Nova and then click on it to expand it?

**Comment by MarByteBeep** [comment]
I think, depending on how it is represented, that could help me a lot!

**Comment by M123-dev** [comment]
FYI @MarByteBeep we are currently rethinking parts of the UI, especially the home and product page and things are looking good that it will include a more central nova score with more details.

It will need a fair share of work though, it's just a mockup in testing right now

**Comment by fabi003** [comment]
I have the same problem.
The amount of UPF in discounters and supermarkets is also raising in Germany as poverty increases.
The best solution is to avoid cheap industry replacements is cooking your own meals and baking your own bread! But who now has the time to do it?
So NOVA score should be the percentage of a product which is known to be traditional food, throwing away everything, where it is unclear, if it is a food industry "improvement" under a clean/"100% natural" looking label. I described the idea in this [forum thread](https://forum.openfoodfacts.org/t/please-add-to-the-nova-4-list/579).
